docs/reference/groups.yml

```
@@ -52,10 +52,9 @@
     "overshell",
   ]
   details: |
-    Delimiters above or below parts of an equation.
+    数式の一部の上部または下部の区切り文字。
```

文字というよりは記号の方がイメージ通りかもしれません。

```suggestion
    数式の一部の上部または下部の区切り記号。
```

---

> over/underlineだけ通常のdelimiterから逸脱している気がします

文の中に入って左右に区切っているいるか、文外に飛び出して上下に区切っているかの違いがあるかと思います。
文中であれば`{`などは「文字」の一つとして考えることはできますが、文外かつそれ単体で使われると「記号」と表現された方がしっくりくるかなといった感覚です。

表記統一に関しては、delimiterをTypstとして常に同じものを指す専門用語と見るか一般的な英単語として見るか、といったところが判断基準になるかと思います。極端な例をあげれば`use`なんかも文脈に応じて適当に訳すことを許容しています。

このコメントをした直後に「確かにどちらとしても解釈できる」と思い直し、レビュー結果をコメントから承認に変更させていただきました。